src/controllers/llmo.js

```diff
@@ -196,13 +205,98 @@ function LlmoController() {
     return ok(config.getLlmoConfig().questions);
   };
 
+  // Handles requests to the LLMO customer intent endpoint, returns customer intent array
+  const getLlmoCustomerIntent = async (context) => {
+    const { llmoConfig } = await getSiteAndValidateLlmo(context);
+    return ok(llmoConfig.customerIntent || []);
+  };
+
+  // Handles requests to the LLMO customer intent endpoint, adds new customer intent items
+  const addLlmoCustomerIntent = async (context) => {
+    const { log } = context;
+    const { site, config } = await getSiteAndValidateLlmo(context);
+
+    const newCustomerIntent = context.data;
```

Comment:
I couldn't see the payload field `newCustomerIntent` in the api spec above. Do we expect client to send a payload like this?

```
{
  "newCustomerIntent" : [ .. ]
}
```